### PR TITLE
Docs: Add documentation and demo for EmptyState and ActionLayout component #DS-1310

### PR DESCRIPTION
**Pull request by curdaj**

### CODE REVIEW

File: packages/web-react/src/components/UNSTABLE_EmptyState/README.md [hunk 1]
```diff
@@ -2,3 +2,79 @@
 
 ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
 Please use it with caution.
+
```
Comment (literat) [inline packages/web-react/src/components/UNSTABLE_EmptyState/README.md:5]: Maybe something like this: `Indicates when there’s no data to show, like when a search has no results.` ?

File: packages/web-react/src/components/UNSTABLE_EmptyState/README.md [hunk 2]
```diff
@@ -2,3 +2,79 @@
...
+<UNSTABLE_EmptyState>
+  <!-- UNSTABLE_EmptyStateSection components or content go here -->
+</UNSTABLE_EmptyState>
+```
+
+### API
+
+The component is based on the [Stack][stack] component and accepts all its properties. For more information, see the [Stack component API][stack-api].
+
+On top of the API options, the components accept [additional attributes][readme-additional-attributes].
```
Comment (literat) [inline packages/web-react/src/components/UNSTABLE_EmptyState/README.md:27]: Maybe these two sentences can be removed.

File: packages/web-react/src/components/UNSTABLE_EmptyState/README.md [hunk 3]
```diff
@@ -2,3 +2,79 @@
...
+<UNSTABLE_EmptyStateSection>
+  <!-- Content goes here -->
+</UNSTABLE_EmptyStateSection>
+```
+
+### API
+
+The component is based on the [Stack][stack] component and accepts all its properties. For more information, see the [Stack component API][stack-api].
+
+On top of the API options, the components accept [additional attributes][readme-additional-attributes].
```
Comment (literat) [inline packages/web-react/src/components/UNSTABLE_EmptyState/README.md:45]: Same here.

File: packages/web-react/src/components/UNSTABLE_EmptyState/README.md (hunk 1 above)
Comment (curdaj) [reply packages/web-react/src/components/UNSTABLE_EmptyState/README.md:5]: I actually used the text from Jobs DS

File: packages/web-react/src/components/UNSTABLE_EmptyState/README.md (hunk 3 above)
Comment (curdaj) [reply packages/web-react/src/components/UNSTABLE_EmptyState/README.md:45]: removed

File: packages/web-react/src/components/UNSTABLE_EmptyState/README.md (hunk 2 above)
Comment (curdaj) [reply packages/web-react/src/components/UNSTABLE_EmptyState/README.md:27]: removed

File: packages/demo/scss/index.scss [hunk 4]
```diff
@@ -61,6 +61,26 @@
     white-space: normal;
 }
 
+.docs-Placeholder {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    padding: 48px 24px;
```
Comment (pavelklibani) [inline packages/demo/scss/index.scss:70]: Rest of padding in this file is in 'rem', shouldn't this be too?

File: packages/demo/scss/index.scss [hunk 5]
```diff
@@ -61,6 +61,26 @@
...
+    text-align: center;
+    gap: 32px;
```
Comment (pavelklibani) [inline packages/demo/scss/index.scss:72]: same as padding, should be this in 'rem'? 🤷

**Review by pavelklibani — APPROVED** [review]

File: packages/demo/scss/index.scss (hunk 5 above)
Comment (curdaj) [reply packages/demo/scss/index.scss]: updated

File: packages/demo/scss/index.scss (hunk 4 above)
Comment (curdaj) [reply packages/demo/scss/index.scss]: updated

**Pull request merged**